LiveSPICEVst/EditorView.xaml.cs

```diff
@@ -95,9 +96,22 @@ private void ShowAboutButton_Click(object sender, RoutedEventArgs e)
         {
             About about = new About() { Owner = Window.GetWindow(this) };
             about.ShowDialog();
-        }
+		}
```

Comment:
It looks like you have a mix of tabs and spaces here, which is rendering differently than it did in your editor (presumably).

I'm pretty sure we use spaces everywhere, can you reformat using spaces only?

Reply:
I have no idea how that happened, since I always use spaces. In any case, thanks for noticing. I've fixed the formatting